WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -121,9 +125,28 @@ - (NSData *)fb_screenshotWithError:(NSError*__autoreleasing*)error
 #endif
 }
 
-- (NSData *)fb_rawScreenshotWithQuality:(NSUInteger)quality rect:(CGRect)rect error:(NSError*__autoreleasing*)error
+- (NSData *)fb_rawScreenshotWithQuality:(NSUInteger)quality rect:(CGRect)rect error:(NSError*__autoreleasing*) error
 {
-  return [XCUIScreen.mainScreen screenshotDataForQuality:quality rect:rect error:error];
+  id<XCTestManager_ManagerInterface> proxy = [FBXCTestDaemonsProxy testRunnerProxy];
+  BOOL fasterAPISupported = [(NSObject *) proxy respondsToSelector:@selector(_XCT_requestScreenshotOfScreenWithID:withRect:uti:compressionQuality:withReply:)];
```

Comment:
I would rather call it inside `dispatch_once` block

Reply:
nice > _XCT_requestScreenshotOfScreenWithID